src/migrations/shared.lua

```diff
@@ -32,8 +34,22 @@ shared.STACKABLES = {
     { item = "processing-unit", icon = nil, stage = 3 },
     { item = "graphite", icon = nil, stage = 1 },
     { item = "fiberboard", icon = nil, stage = 1 },
-    { item = "pcb1", icon = nil, stage = 1 }
-  }
+    { item = "pcb1", icon = nil, stage = 1 },
+    { item = "capacitor1", icon = nil, stage = 1 },
+    { item = "inductor1", icon = nil, stage = 1 },
+    { item = "resistor1", icon = nil, stage = 1 },
+    { item = "optical-fiber", icon = nil, stage = 2 },
+    { item = "resistor2", icon = nil, stage = 2 },
+    { item = "inductor2", icon = nil, stage = 2 },
+    { item = "capacitor2", icon = nil, stage = 2 },
+    { item = "resistor3", icon = nil, stage = 3 },
+    { item = "inductor3", icon = nil, stage = 3 },
+    { item = "capacitor3", icon = nil, stage = 3 }
+  },
+  pyrawores = {
+    { item = "glass", icon = nil, stage = 1 },
+    { item = "limestone", icon = nil, stage = 1 }
```

Comment:
Limestone is part of pycoalprocessing, not pyrawores

Reply:
do i move it ?

Reply:
i don't understand your question.

Reply:
already moved to py coal

Reply:
yes, i saw it and resolved this discussion afterwards.